libs/blocks/global-navigation/global-navigation.js

```diff
@@ -487,7 +487,7 @@ class Gnav {
     const getConfiguration = () => ({
       target: this.blocks.universalNav,
       env: environment,
-      locale,
+      locale: locale === 'no_NO' ? 'nb_NO' : locale,
```

Comment:
I would rather have the condition outside of `getConfiguration` to make sure the value is consistent across code. With this current setup, `nb_NO` would be passed to Universal Navigation, but `no_NO` would get passed if we use `locale` in a different place.

Reply:
Made the change 👍